src/main/java/se/sundsvall/relations/api/model/RelationType.java

```diff
@@ -0,0 +1,19 @@
+package se.sundsvall.relations.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder(setterPrefix = "with")
+@Schema(description = "Type of relation")
+public class RelationType {
+
+	@Schema(description = "Name of type", example = "DUPLICATES")
```

Comment:
Either set all @NotBlank/NotNull fields to requiredMode = REQUIRED or verify if these are set to required automatically, based on the constraint annotations.